test/parallel/test-handle-wrap-isrefed.js

```diff
@@ -110,21 +110,4 @@ const dgram = require('dgram');
 }
 
 
-// timers
-{
-  const timer = setTimeout(() => {}, 500);
-  timer.unref();
-  strictEqual(Object.getPrototypeOf(timer._handle).hasOwnProperty('hasRef'),
-              true, 'timer_wrap: hasRef() missing');
-  strictEqual(timer._handle.hasRef(),
-              false, 'timer_wrap: unref() ineffective');
-  timer.ref();
-  strictEqual(timer._handle.hasRef(),
-              true, 'timer_wrap: ref() ineffective');
-  timer._handle.close(common.mustCall(() =>
-    strictEqual(timer._handle.hasRef(),
-                false, 'timer_wrap: not unrefed on close')));
-}
```

Comment:
This test-case should be rewritten, not removed. `hasRef` here exists for diagnostic tooling purposes.

Reply:
I'm not certain how you envision this being rewritten? There's no longer a handle on the `timer`. We already have a test for TimerWrap separately.

Reply:
Fetching active handles should do, I think? Might be a bit to walk it to find the TimerWrap though. The other option is to export the lists object from `internal/timers` and then import that here.

Reply:
Relevant: my next PR removes TimerWrap altogether in favour of something more akin to how Immediates work. (It's already done, waiting on this one to land.) This test would be very short-lived...

Reply:
Won't that still use a uv handle with the possibility of being unrefed? If so, what is the difference - it should probably has a `hasRef()` anyways for the same reasons.

Reply:
It won't use the C++ HandleWrap anymore so it wouldn't be in getActiveHandles. Right now most diagnostics tools have to filter out TimerWrap manually because it's not very useful. For example, why-is-node-running filters it out from its reports.

(It'll work exactly like Immediates where the libuv internals aren't unnecessarily exposed.)

Reply:
Same as above, have a look at https://github.com/apapirovski/node/commit/d8f075da7b8129f59b1d537b6475394425b81c3e for more info.

Reply:
I still want this for diagnostic tooling if possible. It should be possible to tell if only unrefed timers exist.

(As relating to the second set of changes, I would imagine `Timer.now()` sees use, but we can discuss at that time.)